src/navigation/stripe_count.cpp

```diff
@@ -0,0 +1,130 @@
+/*
+ * Author:Justus Rudolph
+ * Organisation: HYPED
+ * Date: 07/03/2020
+ * Description: Stripe counter class for navigation.
+ *
+ *    Copyright 2019 HYPED
+ *    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ *    except in compliance with the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ *    either express or implied. See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <algorithm>
+#include "navigation/stripe_count.hpp"
+
+namespace hyped {
+namespace navigation {
+
+StripeHandler::StripeHandler(Logger& log, Data& data, NavigationType& displ_unc,
+  NavigationType& vel_unc, NavigationType stripe_dist)
```

Comment:
```suggestion
StripeHandler::StripeHandler(Logger& log, Data& data, NavigationType& displ_unc,
                             NavigationType& vel_unc, NavigationType stripe_dist)
```
I didn't link [constructor initialiser lists](https://hyp-ed.github.io/styleguide/#Constructor_Initializer_Lists) properly before but that's the part you got right funnily enough :D